src/main/java/net/glowstone/command/minecraft/BanListCommand.java

```diff
@@ -0,0 +1,86 @@
+package net.glowstone.command.minecraft;
+
+import org.bukkit.BanEntry;
+import org.bukkit.BanList;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.defaults.VanillaCommand;
+import org.bukkit.util.StringUtil;
+
+import java.util.*;
+
+public class BanListCommand extends VanillaCommand {
+
+    private static final List<String> BAN_TYPES;
+
+    private static final int BAN_TYPES_SIZE;
+
+    static {
```

Comment:
Is there a reason for a static initialization block here? I think you could just to:

```java
private static final List<String> BAN_TYPES = Arrays.asList("ips", "players");
```

No need to store `BAN_TYPES_SIZE` in a variable, just use `BAN_TYPES.size()` when needed.